pkg/execution/state/redis_state/queue_processor.go

```diff
@@ -42,6 +57,10 @@ func (q *queue) Enqueue(ctx context.Context, item osqueue.Item, at time.Time) er
 		queueName = &name
 	}
 
+	go q.scope.Tagged(map[string]string{
+		"kind": item.Kind,
+	}).Counter("queue_items_enqueued_total").Inc(1)
```

Comment:
Nit: Can all of these strings be consts?

Reply:
Yep, good idea!  I'll push out a commit to do that.